.travis.yml

```diff
@@ -1,10 +1,26 @@
 language: python
 sudo: false
 
-python:
-  - "2.7"
-  - "3.5"
-  - "3.6"
+matrix:
```

Comment:
is adding time to the travis build something we're worried about?

Reply:
Possibly, any thoughts on this @twiecki, @richafrank?

Reply:
I'm not worried about it.

Reply:
Each job in the matrix would still take the same amount of time, so we wouldn't suffer any new timeouts. I definitely think we should test an environment that matches what we test for zipline; the others I'm less concerned about. I think we have 4 workers - What's the wall time for a build now?

Is it possible for us to specify the factor, so travis [expands](https://docs.travis-ci.com/user/customizing-the-build#Build-Matrix) the matrix itself?

Reply:
For the last build on this branch it was 20-30 minutes per job, a bit over an hour total.

Reply:
Also yeah looks like we should be able to have travis do the expansion, I'll take a look at that

Reply:
An hour sounds kinda painful to me, but as consumers, what do you all think?